gym_cap/envs/agent.py

```diff
@@ -28,6 +30,7 @@ def __init__(self, loc, map_only, team_number):
         self.range = UGV_RANGE
         self.a_range = UGV_A_RANGE
         self.air = False
+        self.memory = np.full_like(self._env, -1)
```

Comment:
I don't think this would run.

Reply:
changed to self.memory = np.array([[-1 for i in range(self.map_size[0])] for j in range(self.map_size[1])])

Reply:
use map_only.shape

Reply:
changed accordingly